flow-libs/semver.js.flow

```diff
@@ -1,29 +1,30 @@
-// flow-typed signature: dc381ee55406f66b7272c6343db0834b
-// flow-typed version: da30fe6876/semver_v5.1.x/flow_>=v0.25.x
+// @flow
```

Comment:
Can these be contributed upstream? Otherwise, can this be moved to `flow-libs` (with our other manually maintained libdefs) instead? 😄

Reply:
-> https://github.com/flow-typed/flow-typed/pull/3674
And I did move that file to flow-libs after modifying it 🙂

Reply:
😄